src/main/java/org/broadinstitute/hellbender/tools/exome/Pulldown.java

```diff
@@ -34,7 +35,8 @@ public Pulldown(final File inputFile, final SAMFileHeader header) {
      * AllelicCounts.  This IntervalList is modifiable and does not change with the state of the Pulldown.   */
     public IntervalList getIntervals() {
         final IntervalList intervals = new IntervalList(header);
-        intervals.addall(getCounts().stream().map(AllelicCount::getInterval).collect(Collectors.toList()));
+        intervals.addall(getCounts().stream().map(AllelicCount::getInterval)
```

Comment:
second map is just to transform SimpleInterval -> Interval.  previously AllelicCount's interval was represented by Interval (since SamLocusIterator wants an IntervalList) so this map wasn't necessary, but i changed it to a SimpleInterval throughout in this commit so that AllelicCount isn't the only class using Interval